packages/webview_flutter/webview_flutter/lib/src/webview_controller.dart

```diff
@@ -405,6 +405,16 @@ class WebViewController {
   ) {
     return platform.setOnScrollPositionChange(onScrollPositionChange);
   }
+
+  /// Whether the vertical scrollbar should be drawn or not.
+  Future<void> setVerticalScrollBarEnabled(bool enabled) {
+    return platform.setVerticalScrollBarEnabled(enabled);
+  }
+
+  /// Whether the horizontal scrollbar should be drawn or not.
+  Future<void> setHorizontalScrollBarEnabled(bool enabled) {
+    return platform.setHorizontalScrollBarEnabled(enabled);
+  }
```

Comment:
Can you also plumb through a `Future<bool> supportsSetScrollBarsEnabled()` method to the platform interface layer that clients can use to detect whether this is safe to call per https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#api-support-queries ? This won't make sense on all platforms, so we should expect that clients will need a support query indefinitely.

Reply:
I'm also assuming you meant for this to return `bool` and not `Future<bool>` too.

Reply:
Oops, I missed this, and didn't catch it in the platform interface review. It should have been `Future<bool>` in case there were a platform that could only determine this at runtime (e.g., based on the OS version).

Not a big deal though as that's an unlikely case to ever hit.